.circleci/config.yml

```diff
@@ -97,7 +97,7 @@ commands:
 jobs:
   validate-code:
     macos:
-      xcode: 15.1 # Specify the Xcode version to use
+      xcode: 16.0.0 # Specify the Xcode version to use
```

Comment:
Updated to Xcode 16, as the latest SDKs include significant updates related to Swift 6 concurrency and align with the Xcode version used locally for development.